pastas/plotting/modelcompare.py

```diff
@@ -907,7 +931,7 @@ def plot(
         grid: bool = True,
         legend: bool = True,
         adjust_height: bool = False,
-        legend_kwargs: Optional[dict] = None,
+        legend_kwargs: Optional[dict] = {},
```

Comment:
I think this should be `None` as default argumnt, not `dict` right? Otherwise ruff and/or codacy will start complaining about mutable default arguments.

Reply:
Oui. good point. And add the if statement again.